collector/meminfo_linux.go

```diff
@@ -89,6 +121,16 @@ func parseMemInfo(r io.Reader) (map[string]float64, error) {
 	for scanner.Scan() {
 		line := scanner.Text()
 		parts := strings.Fields(string(line))
+		key := ""
+
+		// /proc/meminfo format:
+		//   <metric name>: <metric val> [kB]
+		// /sys/devices/system/node/node%d/meminfo format:
+		//   Node <node number> <metric name>: <metric val> [kB]
+		if len(parts) >= 4 {
+			key = parts[0] + parts[1] + "_"
```

Comment:
@brian-brazil Looks like I don't fully understand the meaning of ConstMetric then.
I assumed it's useful when you have a constantly changing set of labels, like in filesystem collector where new mount points may come and go at any time. And this allows to not store stale metrics in memory and reduce overall memory consumption.

In this case, the set of metric is constant (fields of meminfo files) and the set of labels is also constant (available NUMA node). So I don't understand how ConstMetric can be helpful here.

Could you please elaborate?